src/io/parsers/VolumeModel.js

```diff
@@ -109,11 +114,42 @@ class VolumeModel {
   }
 
   _getVolumeInfo() {
-    return _.pick(this._header, ['dmean', 'dmin', 'dmax', 'sd']);
+    const volInfo = _.pick(this._header, ['dmean', 'dmin', 'dmax', 'sd', 'delta']);
+    volInfo.obtuseAngle = [];
+    this._header.angles.forEach((angle, i) => { volInfo.obtuseAngle[i] = (Math.PI / 2) - angle > 0 ? 0 : 1; });
```

Comment:
If you'd like to use an array, the following is another way to achieve the same result:
```suggestion
    volInfo.obtuseAngle = this._header.angles.map(angle => Number(angle >= (Math.PI / 2)));
```

If three.js would allow it, we could use:
```suggestion
    volInfo.obtuseAngle = this._header.angles.clone();
    volInfo.obtuseAngle.stepScalar(Math.PI / 2); // step() and stepScalar() are missing, unfortunately
```